instance/instance.go

```diff
@@ -109,6 +109,66 @@ func (s *Store) Add(w http.ResponseWriter, r *http.Request) {
 	log.Debug("add instance", log.Data{"instance": instance})
 }
 
+// UpdateDimensionLabelDesc updates label and/or description for a specific dimension within an instance
+func (s *Store) UpdateDimensionLabelDesc(w http.ResponseWriter, r *http.Request) {
+
+	vars := mux.Vars(r)
+	ID := vars["id"]
+	dimension := vars["dimension"]
+
+	// Get instance
+	instance, err := s.GetInstance(ID)
+	if err != nil {
+		log.Error(err, nil)
+	}
+
+	// Early return if instance is already published
+	if instance.State == "published" {
+		w.WriteHeader(http.StatusForbidden)
+		return
+	}
+
+	// Read and unmarshal request body
+	bytes, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		log.Error(err, nil)
```

Comment:
Same here if we get an error do we want to log out the error & message for context and then return an error status?